pkg/storage/ingest/reader.go

```diff
@@ -721,27 +719,8 @@ func (r *PartitionReader) waitReadConsistency(ctx context.Context, withOffset bo
 func (r *PartitionReader) pollFetches(ctx context.Context) (result kgo.Fetches) {
 	defer func(start time.Time) {
 		r.metrics.fetchWaitDuration.Observe(time.Since(start).Seconds())
-		result.EachRecord(func(record *kgo.Record) {
-			r.metrics.fetchedBytes.Add(float64(len(record.Value)))
-		})
 	}(time.Now())
-
-	f := r.client.PollFetches(ctx)
-	for fIdx, fetch := range f {
-		for tIdx, topic := range fetch.Topics {
-			for pIdx, partition := range topic.Partitions {
-				afterConsumed := len(partition.Records)
-				for i, record := range partition.Records {
-					if record.Offset > r.consumedOffsetWatcher.LastConsumedOffset() {
-						afterConsumed = i
-						break
-					}
-				}
-				f[fIdx].Topics[tIdx].Partitions[pIdx].Records = partition.Records[afterConsumed:]
-			}
-		}
-	}
-	return f
```

Comment:
this is no longer necessary since we're using the same fetcher implementation for startup fetching and regular fetching. Previously we needed a way to synchronise the two - make sure the latter picked up where the former left off.